kubernetes_state/datadog_checks/kubernetes_state/data/auto_conf.yaml

```diff
@@ -25,6 +25,6 @@ instances:
     # will be attached to the host running KSM)
     # hostname_override: true
     #
-    # Send (deprecated) pod.phase service checks (default true, will be turned off in future versions)
+    # Send (deprecated) pod.phase service checks (default false)
     # We recommend using the corresponding gauges instead, which benefit from historical data unlike service checks
-    # send_pod_phase_service_checks: false
+    # send_pod_phase_service_checks: true
```

Comment:
was this correct before, actually?

Reply:
The default is `false` in the code, so I set the commented one to `true`, this way users can just uncomment this line if they want it. That's the convention I've been using so far, have we decided on something different?

Reply:
IIRC with the new standard @l0k0ms is setting the default value in the yaml